content/blog/2015/06/2015-06-18-status-update-2/index.md

```diff
@@ -67,19 +67,19 @@ count is low, but it will only remain that way if you don't get requesting!
 
 This section will be updated in due course to reflect the *new* bug tracker.
 
-- [Git Repositories](https://git.solus-project.com/?s=idle)
-- [Build System](https://build.solus-project.com)
-- [Working with the new infrastructure](https://git.solus-project.com/common/about/)
-- [Current Bug Tracker](https://dev.solus-project.com)
```

Comment:
I think base dev tracker links like this should link to the GitHub repository root instead.